src/test/ui/consts/const-float-classify.rs

```diff
@@ -53,6 +53,9 @@ impl const PartialEq<NonDet> for bool {
     fn eq(&self, _: &NonDet) -> bool {
         true
     }
+    fn ne(&self, _: &NonDet) -> bool {
```

Comment:
Why add these?

Reply:
because we deny using the default functions, for now.

in the future, it might verify if the default function is `const`. but for now, it denies if a default function is used

Reply:
Oh I see. Whoops.